powershell-analyzer/README.md

```diff
@@ -0,0 +1,71 @@
+# powershell-analyzer
+
+Lint PowerShell scripts and modules using [PSScriptAnalyzer](https://docs.microsoft.com/en-us/powershell/module/psscriptanalyzer). This can be run locally or executed in GitHub Actions.
+
+## Run Locally
+
+```bash
+.\analyze.ps1 -Directory /folder-or-file
+```
+
+### Options
+
+| Parameter     | Description                                                                                                                                                       |
+| ------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Directory     | Folder or Path to run analysis against. Required.                                                                                                                 |
+| SaveToFile    | Save to file, or print to console. Default: Save to file                                                                                                          |
+| ResultsPath   | Path to save the results. Default "./results.sarif"                                                                                                               |
+| IncludedRules | [List of rules](https://docs.microsoft.com/en-us/powershell/utility-modules/psscriptanalyzer/rules/readme?view=ps-modules) that should be included to analysis.   |
+| ExcludedRules | [List of rules](https://docs.microsoft.com/en-us/powershell/utility-modules/psscriptanalyzer/rules/readme?view=ps-modules) that should be excluded from analysis. |
+
+### Result Formats
+
+- [SARIF](https://sarifweb.azurewebsites.net/) reports
+- Print to console
+
+## Run in Action
+
+## Example usage
+
+Minimal uses expression to use this action:
+
+``` yaml
+- uses: ./powershell-analyzer
+  name: Lint PowerShell Scripts
+  id: powershell-analyzer
+```
+
+Example for calling this action from a different repository:
+
+```yml
+- uses: Ed-Fi-Alliance-OSS/Ed-Fi-Actions/powershell-analyzer@latest
+  name: Lint PowerShell Scripts
+  id: powershell-analyzer
+```
+
+❗ In this one case, it is appropriate to use a tag ("latest") instead of a
```

Comment:
Now we have multiple cases 😆. The rest of the comment below isn't applicable - it only applied to the repo scanner. We could use commit hash here. On the other hand, this is our own repo: is there any risk to using `@latest` with our own repository?

```suggestion
❗ In this case, it is appropriate to use a tag ("latest") instead of a
```

Reply:
I agree, there's no risk on using @latest here. I will remove this comment